pytket/phir/api.py

```diff
@@ -91,6 +93,19 @@ def qasm_to_phir(
     :param circuit: Circuit object to be converted
     :param qtm_machine: (Optional) Quantinuum machine architecture to rebase against
     :param tket_optimization_level: (Default=0) TKET circuit optimization level
+    :param wasm_bytes (Optional) WASM as bytes to include as part of circuit
     """
-    circuit = circuit_from_qasm_str(qasm)
+    circuit: Circuit
+    if wasm_bytes:
```

Comment:
On Windows I am getting "PermissionError: [Errno 13] Permission denied" for the wasm temporary file.

Reply:
I have had issues in the past with creating temporary files where the file was getting deleted after I wrote to it without the right settings. I don't know if that is what is going on here.

Reply:
If it is any help, this is what worked for me in the past:
```python
with tempfile.NamedTemporaryFile(mode="w+",suffix=".qasm", delete=False) as temp_file:
    temp_file.write(program)
    temp_file.flush()
    temp_file_path = Path(temp_file.name)
    program = qasm2pytket_handler(temp_file_path, machine)
temp_file_path.unlink()
```